.github/workflows/liam-erd-sample-e2e_tests.yml

```
@@ -0,0 +1,142 @@
+              "should_run=false"
+            end
+
+          # Same as 'echo "should_run=xxxx" >> $GITHUB_OUTPUT'
+          output_file = ENV.fetch("GITHUB_OUTPUT")
+          File.open(output_file, "a") do |file|
+            file.puts result
+          end
+
+      - name: Setup pnpm
+        if: steps.check.outputs.should_run == 'true'
+        uses: actions/pnpm-setup
```

actionlint (1.7.7)

specifying action "actions/pnpm-setup" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)